config/master/manifests.jsonnet

```diff
@@ -144,7 +144,7 @@ local awsnode = {
           containers_:: {
             awsnode: {
               image: "%s/amazon-k8s-cni:%s" % [$.ecrRepo, $.version],
-              imagePullPolicy: "Always",
+              imagePullPolicy: "IfNotPresent",
```

Comment:
Another option is to set this conditionally in jsonnet:
```jsonnet
imagePullPolicy: if $.version == "latest" then "Always" else "IfNotPresent",
```

Tbh, I've found that more trouble than it's worth in the past (eg: a naive kubectl patch might change the image, but forget to update the imagePullPolicy) so again I suggest just removing the field entirely and using the k8s dynamic default.

Reply:
yeah, you are totally right, the behavior of k8s is by default "IfNotPresent". So we proceed with removing the fields and just mention the condition in the jsonnet manifest. I will update the PR accordingly.